### PR TITLE
Add support for shallow routing

### PR DESCRIPTION
**Pull request by vajajak**
- Introduces a simple option to disable loading indicator when navigating shallowly on one page
- Fixes https://github.com/apal21/nextjs-progressbar/issues/33

### CODE REVIEW

**Comment by apal21** [comment]
Cool. I'll check this over the weekend and merge it.

File: README.md [hunk 1]
```diff
@@ -24,12 +24,14 @@ And for rendering add `<NextNProgress />` inside `Container` component.
...
   stopDelayMs={200}
   height={3}
+  showOnShallow={true}
 />
 ```
 * `color`: to change the default color of progressbar. You can also use `rgb(,,)` or `rgba(,,,)`.  
 * `startPosition`: to set the default starting position : `0.3 = 30%`.
 * `stopDelayMs`: time for delay to stop progressbar in `ms`.  
 * `height`: height of progressbar in `px`.  
+* `showOnShallow`: boolean, choose whether you want the progressbar to display if you're navigating shallowly inside one page. Learn more about shallow routing [in Next.js docs](https://nextjs.org/docs/routing/shallow-routing).
```
Comment (apal21) [inline README.md:34]: I think this should be something like:

```
`showOnShallow`: You can choose whether you want the progressbar to be displayed if you're using shallow routing. It takes a boolean. Learn more about shallow routing [in Next.js docs](https://nextjs.org/docs/routing/shallow-routing).
```

**Comment by apal21** [comment]
Hey @vajajak, I checked the code. It is working fine. I've added a comment in the `README.md` file. I'll merge this once it is fixed!

**Comment by vajajak** [comment]
Hi, ok let's go with that :) Please have a look now.

**Pull request merged**